ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/TestCaseTest.java

```diff
@@ -94,8 +92,11 @@ public void init() {
   @SuppressWarnings("unchecked")
   public void shouldProcessInputRecords() {
     // Given:
-    final TopologyTestDriverContainer topologyTestDriverContainer = getSampleTopologyTestDriverContainer();
-
+    final TopologyTestDriverContainer topologyTestDriverContainer = TopologyTestDriverContainer.of(
```

Comment:
why not use `getSampleTopologyTestDrivenContainer()`?

Reply:
This one used `topic` and does not create a new topic.